src/ol/layer/layerbase.js

```diff
@@ -35,6 +35,7 @@ ol.layer.LayerProperty = {
  *            saturation: number,
  *            sourceState: ol.source.State,
  *            visible: boolean,
+ *            unmanaged: (boolean|undefined),
```

Comment:
We could also decide to always/systematically define it, but it's ok like this as well.